pkg/instrumentation/podmutator_test.go

```diff
@@ -9,6 +9,8 @@ import (
 	"os"
 	"testing"
 
+	"github.com/aws/amazon-cloudwatch-agent-operator/internal/manifests/collector/adapters"
```

Comment:
why did this get moved? can you run the formatter and goimports?

Reply:
done